app/src/main/java/com/cpr3663/cpr_scouting_app/Logger.java

```diff
@@ -338,12 +338,21 @@ public void LogData(String in_Key, String in_Value) {
         match_log_data.add(new Pair<>(in_Key, in_Value.trim()));
     }
 
+    // Member Function: Log a non-time based event - just store this for later.
```

Comment:
This description does not make sense.

Reply:
i fixed it in my next upcoming checkin already.

Reply:
i fixed it in my next upcoming checkin already.